static/schemas/config.evm.json

```diff
@@ -0,0 +1,470 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Envio Config Schema",
+  "description": "Schema for a YAML config for an envio indexer",
+  "type": "object",
+  "properties": {
+    "description": {
+      "description": "Description of the project",
+      "type": [
+        "string",
+        "null"
+      ]
+    },
```

Comment:
Maybe we can make the JSONSchema be generated with the same fields order as listed on a struct. It would be nice to have the `name` property first in the explorer.

Reply:
I can see there's `preserve_order` feature flag https://docs.rs/schemars/latest/schemars/index.html#feature-flags

Reply:
Ah nice! I'll investigate now thanks!